### ISSUE TITLE
Version update coming?

### ISSUE THREAD

**Issue by jweidner-mbible** [xamelon/flutter_pay#6]
I noticed the 0.9.1 version on pub.dev/packages/flutter_pay seems to not be up to date with the latest code here.  For example the makePayment method seems to be renamed requestPayment.  Do you plan to update the pub.dev package soon?

**Comment by ArturMerlin** [comment]
Yes, last pull request updated version of plugin to 0.10.0, but version on pub.dev - 0.9.1.  @xamelon update, please, plugin on pub.dev
